ironfish/src/consensus/consensus.ts

```diff
@@ -65,6 +65,11 @@ export const GRAFFITI_SIZE = 32
  */
 export const IRON_FISH_YEAR_IN_BLOCKS = (365 * 24 * 60 * 60) / TARGET_BLOCK_TIME_IN_SECONDS
 
+/**
+ * Max transactions allowed per block
+ */
+export const MAX_TRANSACTIONS_PER_BLOCK = 300
```

Comment:
This will cause the network to halt, if you look at our block explorer all our blocks have a max of 301.